pkg/tecdsa/tecdsa.go

```diff
@@ -0,0 +1,19 @@
+package tecdsa
+
+import "time"
+
+const (
+	DefaultPreParamsPoolSize              = 50
```

Comment:
That's a lot. Could we reduce the value? How often do we expect a pre param will be popped from the pool?
Generation is very CPU consuming and generating so many numbers may take a long time and cause a headache on production machines.

Reply:
This is an absolute minimum to run two clients locally. The group has 100 members and assuming even distribution (it is usually not even), we need to have 50 members per node.

This PR is just about allowing to configure these values and I used the same value as it was before (see `pkg/tbtc/node.go`). In a separate set of PRs, we'll have to rework the pre-params generation process to generate way more pre-params but only when none of the protocols are currently executing.